include/zephyr/drivers/i3c.h

```diff
@@ -56,19 +56,71 @@ extern "C" {
  *   - 0: No Limitation
  *   - 1: Limitation obtained via GETMXDS CCC.
  */
+
```

Comment:
for each group of macros defining the various bits of a register, you may want to consider putting them in `@name { ... }` sections

```c
/**
 * @name Bus Characteristic Register (BCR) fields.
 * @{
 */

...

/** @} */
```

Reply:
Moved BCR and DCR under named section.